modules/app.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please do not disable eslint, try to fix the issues that eslint is poining on. For example `/* eslint-disable import/prefer-default-export */ `can be fixed by:

`export default  MainUI ;`
`import  MainUI  from './modules/app.js';`